lxd/storage_zfs.go

```diff
@@ -1842,75 +1842,73 @@ func (s *storageZfs) zfsPoolCreate() error {
 
 	// Create default dummy datasets to avoid zfs races during container
 	// creation.
-	err := s.zfsPoolVolumeCreate("containers")
-	if err != nil {
-		return err
-	}
-
-	err = s.zfsPoolVolumeSet("containers", "mountpoint", "none")
+	poolName := s.getOnDiskPoolName()
+	dataset := fmt.Sprintf("%s/containers", poolName)
+	fixperms := shared.VarPath("storage-pools", s.pool.Name, "containers")
+	msg, err := zfsPoolVolumeCreate(dataset, fmt.Sprintf("mountpoint=%s", fixperms))
```

Comment:
So why do we set this one? I thought every container filesystem we create has a mountpoint= directly set on it, making it unnecessary for us to set a mountpoint on the parent.